docs/faqs.asciidoc

```diff
@@ -33,13 +32,13 @@ The cost of deploying from the Marketplace will vary depending on the cluster
 topology that you decide to deploy and the VM SKUs that you choose for each of the
 VMs within the cluster. The cost will also vary depending on which Azure region
 you decide to deploy to. Because there are a number of variables involved, a cost
-needs to be calculated on a case-by-case basis. At a minimum, you will pay for
+needs to be calculated on a case-by-case basis. **At a minimum**, you will pay for
 all Azure resources deployed by the template. The
 {azurecalculator}[Azure pricing calculator] can help in providing a good estimate
 for this. In addition to the cost of Azure resources,
 {subscriptions}[Elastic Stack Gold, Platinum and Enterprise features] require a
 license to be purchased, the cost of which is dependent on a number of factors.
-Please get in touch with us to discuss.
+{contact}[Please get in touch with us to discuss].
```

Comment:
Call to action; website link? e-mail address?

Reply:
It's an attribute defined in  `docs/azure-marketplace.asciidoc`